test.py

```diff
@@ -275,6 +276,9 @@ def clean_up_tests():
         "processed_data.nwb",
         "raw_data.nwb",
         "scratch_analysis.nwb",
+        # "sub-P11HMH_ses-20061101_ecephys+image.nwb",  # TODO cannot delete this file on windows for some reason
```

Comment:
Often when delete on Windows fails it means that the file was not closed properly, i.e., some process still indicated that it is using the file.